last_branch.txt

```diff
@@ -0,0 +1,244 @@
+feature/GREAT-761/GREAT-1022/alexsherstinsky/rule_based_profiler/data_assistant/data_assistant_result/include_detailed_rule_statistics_in_jupyter_notebook_repr_output-2022_06_23-182
```

Comment:
Please remove this - `ge_releaser` stages and commits all changes

Reply:
@cdkini Did I delete it successfully?

Reply:
👍🏽